internal/dataplane/parser/translators/ingress.go

```diff
@@ -62,7 +63,12 @@ func TranslateIngresses(
 // Ingress Translation - Private Consts & Vars
 // -----------------------------------------------------------------------------
 
-var defaultHTTPIngressPathType = netv1.PathTypeImplementationSpecific
+var (
+	defaultHTTPIngressPathType = netv1.PathTypeImplementationSpecific
+
+	uriCapturesPrefix = []rune("$(uri_captures[")
+	uriCapturesSuffix = []rune("])")
```

Comment:
You can make them as regular const strings and use [WriteString](https://pkg.go.dev/strings#Builder.WriteString) (as I suggested to use `strings.Builder`, see next comments)